packages/block-library/src/image/index.js

```diff
@@ -158,7 +158,7 @@ export const settings = {
 						selector: 'img',
 					},
 					caption: {
-						type: 'array',
+						type: 'rich-text',
 						// To do: needs to support HTML.
```

Comment:
Should we remove this comment now?

Reply:
Why is it wrong? If you have an attribute source you can have different types as well, that's why you can declare a different type?